### ISSUE TITLE
Image scale evolved CPPNs

### ISSUE THREAD

**Issue by schrum2** [schrum2/MM-NEATv2#87]
I removed the large French flag from the repository because it should be completely unnecessary. The tiny one should be enough. One of the cool things about CPPNs is that they encode information with infinite resolution. This means that, in theory, if we can successfully evolve a CPPN to generate the tiny French flag, we should be able to take that same CPPN and query it's coordinates with a different image size, and get the same image! Verify that this is the case by writing some code to test it.

I think the best way to do this is with some command line parameters that override the specified size being used to draw a CPPN image. This way, if the commands from postBestObjectiveWatch are used with these additional overrides, we can scale up our image results.

**Comment by leg2015** [comment]
Added 4 new command line parameters that allow for a default image size and for the image size to be dictated as a command line parameter also. I also added a test in the main method of the image match task to create two random images with the same seed but different sizes to see if it would work, but it hasn't yet. Might be the recurrent networks causing that problem but I'm going to go ahead and close this issue because the recurrent links problem is not part of this issue.
